traitlets/traitlets.py

```diff
@@ -2263,12 +2263,21 @@ def validate(self, obj, value):
                 return value
         self.error(obj, value)
 
-    def info(self):
+    def _choices_str(self, as_rst=False):
+        """ Returns a description of the trait choices (not none)."""
+        choices = self.values
+        if as_rst:
+            choices = '|'.join('``%r``' % x for x in choices)
+        else:
+            choices = repr(list(choices))
+        return choices
+
+    def info(self, as_rst=False):
```

Comment:
Adding an `as_rst` arg is a backward-incompatible change because user traits can have defined `.info()` accepting no arguments, which must not result in any errors.

Maybe better to define a new `.info_rst()` method that calls `.info()` by default.